indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -4,6 +4,7 @@
     $(document).ready(function() {
         setupSurveyScheduleWindows();
         setupQuestionWindows();
+        setupSubmissionButtons();
```

Comment:
do we really need to initialize this on page load whenever this file is included? I'd rather for for `<script>setupSubmissionButtons();</script>` in the template of the page where we need it.